src/containers/Search/index.tsx

```diff
@@ -56,6 +59,12 @@ const Search = (homeProps: IHomeProps) => {
       })
 
       if (res.data.nodes.length > 0 && cy) {
+        if (ID_REGEX.test(query)) {
```

Comment:
really i think we can have a different endpoint for search by id; also the graph link should almost always start this way

Reply:
If that's not something you've done in the latest PRs, I'll update its endpoint to expect only ID and remove the search bar from the graph in FE